### DISCUSSION TITLE
Post your Memtest86+ v6 screenshots here!

### DISCUSSION THREAD

**Discussion by x86fr** [memtest86plus/memtest86plus#20] [Show and tell]
Just for fun, let's post Memtest86+ v6 screenshot here, so we can keep a trace of the evolution over time 😊

All screenshots welcome but strange/rare hardware particularly appreciated!

Let's start with some DDR3/DDR4/DDR5 combo

![IMG_2358](https://user-images.githubusercontent.com/38105886/159831034-68759e38-87cf-4330-8614-963c1f4c5889.jpg)

![IMG_2359](https://user-images.githubusercontent.com/38105886/159831046-46ca7841-b55d-4589-8a40-92bae8afcaed.jpg)

![IMG_2360](https://user-images.githubusercontent.com/38105886/159831052-5d9a6b95-81a0-4ab4-8e72-4d813da137b6.jpg)

**Comment by martinwhitaker** [comment]
I frequently use QEMU for testing, so here's a screenshot of that:
![memtest-qemu.png](https://user-images.githubusercontent.com/4000916/159880041-1189b3d0-8ec4-474d-b10b-81f647b7b110.png)

**Reply by x86fr** [reply]
Here is some REALLY exotic hardware 😱

![image](https://user-images.githubusercontent.com/38105886/160165552-35ae6dc8-538c-4ad0-9ede-110987fdb6e2.jpeg)

Unfortunately, SMP is not detected.

**Reply by x86fr** [reply]
More uncommon hardware: Zhaoxin CPU (now supported)
![image](https://user-images.githubusercontent.com/38105886/160186168-95df6ba4-f44c-4723-9e35-82cb5b8df72e.jpeg)

**Comment by corvus1** [comment]
![IMG_20220622_052011](https://user-images.githubusercontent.com/4068104/174923452-61c64ab3-8e54-4574-b989-4be7ac7d30cd.jpg)

Motherboard PC Chips M789CG-2000

Thought I'd entertain you with exotic hardware, but it seems like it has no RAM! :-D 

By the way, do not remove -march=i586 from CFLAGS, or else there'll be no more screenshots from that board. Or maybe could add it myself, ofc...

**Reply by debrouxl** [reply]
https://valid.x86.fr/x7uu6c and https://www.driverscape.com/manufacturers/ecs/laptops-desktops/m789cg/47455 consistently indicate that this MB has a VIA chipset, which wasn't unexpected with that processor anyway.
By now, https://github.com/memtest86plus/memtest86plus/blob/main/system/smbus.c contains support for nearly all models of I2C/SMBus controllers for which CPU-Z reports have been created recently (#86), as well as PCI IDs + Linux driver name indication for the ones which aren't supported (yet). VIA chipsets, among which the VT8235 supported by Linux "i2c-viapro" is, belong to the latter category :)

**Reply by jonathan-teh** [reply]
Cache detection for VIA Samuel 2 was initially added in #259 then fixed in #341, and VT8235 SMBus support was added in memtest86plus/memtest86plus@10e843560494a8d so everything should work now.

**Comment by 01e3** [comment]
Asus P6T Deluxe V2 (Intel X58) w/Intel Xeon X5660 and 4xPC3L-12800R (ECC RDIMM) memory - M392B1G73DB0

![P6TDeluxeV2](https://user-images.githubusercontent.com/105136494/174960515-6cc82921-2d3e-4bd6-b483-8c37f077ebb2.png)

Note the "pass completed" message is still misaligned. Other then that, everything works but will file a separate bug for USB taking almost 1m to detect when an USB hub from the monitor is connected. Keyboard works however. :)

**Comment by 01e3** [comment]
[FIC VA-503+](https://www.anandtech.com/show/150) (VIA Apollo MVP3 ) w/ AMD K6-2 3D 450MHz CPU.

The motherboard has 1024KB cache installed, but it is not detected by Memtest so no benchmark.

Also, no SPD. ;)


![VA-503+](https://user-images.githubusercontent.com/105136494/175019533-b0baa45c-fe3d-4d5d-a24a-0d704977a93d.png)

**Reply by x86fr** [reply]
Such old chipsets requires specific proprietary code to detect the external cache. Only the most common ALi ALADDiN V is supported.
Same for SPD. A new code must be wrote and it's really not on top of the list right now ;)

**Reply by 01e3** [reply]
Completely understood, was just trying to follow the spirit of this thread. :)

**Reply by debrouxl** [reply]
Thanks for these tests :)
Yeah, support for both detection of external L2 cache on that platform and VIA's SMBus controllers would be best contributed by people interested in those 20+-year-old pieces of hardware.
Like the ECS M789CG above, this MB's chipset is from VIA ( https://www.ultimateretro.net/fr/motherboards/8482 , etc.), and the SMBus controller may be supported by e.g. Linux's "i2c-viapro" driver.

**Reply by corvus1** [reply]
The VIA C3 (Samuel) that I posted above has integrated CPU cache, both L1 and L2. To make thing funnier, if you follow the link, CPU-Z claims, it has 16 megabytes (!) of L2 cache, VIA must've been doing something right, because my ryzen 2700 has 16MB of L3. And CPU-Z is not the only one reporting that.
 
![IMG_20211011_202733](https://user-images.githubusercontent.com/4068104/175027519-0ae9dfec-3585-4afb-9308-8cfdf7de69cd.jpg)

**Reply by x86fr** [reply]
Total mess but not uncommon with VIA CPUs :)

CPUID on VIA C3/C7 is writable, so the BIOS can wrongly set any value anywhere.

**Reply by jonathan-teh** [reply]
Cache detection for VIA Apollo [M]VP3 is being added in #356.

**Comment by 01e3** [comment]
Today something even more ancient... 

[Shuttle HOT-419](https://www.ultimateretro.net/en/motherboards/4439) with AMD 804086DX4-120 and 3x8MB=24MB RAM. No PCI. ISA & VLB.

And yes, the motherboard has 256KB of cache. ;)

![486dx4-120](https://user-images.githubusercontent.com/105136494/175603857-d863449b-1ca2-442a-9e8e-83989b0feabf.png)

To my surprise, I did not need to re-compile memtest as -march=i486, the stock (32bit ofc) image just worked. 

Now some ugly (but I guess expected) issues:
 - there is no clock,
 - testing memory never starts unless "Enter" is pressed,
 - testing memory never completes - hangs on the bit fade test,
 - reboot does no work - hangs with "Rebooting...",
 - it is very slow, about 1h to reach the bit fade test with 24MB of ram.

**Reply by corvus1** [reply]
that's good to know, I have a 486DX2-66, I was thinking of testing that, but I thought I'd have to change CFLAGS for it to even start. 
But slow? 1h is not slow. 2 days would be slow.

**Comment by cheapie** [comment]
I was told strange hardware was particularly appreciated, so...

DM&P Vortex86DX3, single-core version, with 1GiB of RAM (ICOP VDX3-6754-1G motherboard)

![Memtest86+ 6.00b2 running on a Vortex86DX3](https://cheapiesystems.com/media/images/IMG_20220704_123346029.jpg)

This is with the official 32-bit image, with no special tricks needed to make it run. The only real issue I spotted was that the cache (should be 32KiB L1i, 32KiB L1d, and 512KiB L2, but it may be lower on this single-core variant) was not detected. The lack of SPD is expected as the RAM is soldered to the board.

-----

SiS 550 (effectively a Rise mP6 core and some SiS chipset on one chip), in a Neoware CA5 thin client:

![Memtest86+ 6.00b2 running on a SiS 550](https://cheapiesystems.com/media/images/IMG_20220704_165932098.jpg)

Cache is again missing (I think it's supposed to have 8KiB of L1i and 8KiB of L1d) and the CPU is just identified as "586" (the CPU reports vendor "SiS SiS SiS " family 5 model 0 stepping 5, but no brand string), but it otherwise seems to work.

-----

Cyrix/National Semiconductor Geode GX1 300MHz in a Neoware Capio 620 thin client:

![Memtest86+ 6.00b2 running on a Geode GX1](https://cheapiesystems.com/media/images/IMG_20220704_184443228.jpg)

I have no idea what the cache is supposed to be on this one, but everything else seemed to work fine.

-----

Transmeta Crusoe TM5700 in an HP t5710 thin client, VIA VT8231 southbridge (northbridge is integrated into the CPU):

![Memtest86+ 6.00b2 running on a Transmeta Crusoe TM5700](https://cheapiesystems.com/media/images/IMG_20220704_191535323.jpg)

I'm starting to see a theme here with the cache... that aside, like all of this other hardware, it ran fine with the official 32-bit image, no weird tricks needed.

**Reply by jonathan-teh** [reply]
Cache detection for Vortex86 was added in #290 and Geode GX1 in #267. Support for SiS 550 and Transmeta can be added, if you're willing to test (CI makes binaries from PRs available for download)?

**Comment by franck-888** [comment]
Laptop Sony Vaio VPCCB3M1E
ca. 2011, bios, no boot capability on gpt partitions
BIOS R2100V2 08/24/2011
Intel Core i3-2330M CPU @ 2.20GHz
8GB RAM DDR3 (2x4GB)
![vaio](https://user-images.githubusercontent.com/107723864/178153710-b7ded528-13a5-4f4e-b718-cab3ab5951b1.png)

**Comment by franck-888** [comment]
Intel Core 2 CPU 6600 originally 2.40GHz overclocked to 3.0GHz, 2 cores
8GB RAM DDR2 (4x2GB)
Motherboard: Asustek Computer Inc P5W DH Deluxe, BIOS 3002 07/22/2010
Build 6.00b3 + up to latest commit 740df34656b122613f3e1f506daff6d16174a6c2
![build](https://user-images.githubusercontent.com/107723864/180635088-8dd1193b-1b29-413e-83ec-dc15c3b42aaa.jpg)

**Comment by corvus1** [comment]
![IMG_20220724_213404](https://user-images.githubusercontent.com/4068104/180659265-f228bc7a-6f81-4dec-9706-4073589cdd32.jpg)

Allow me to introduce my Toshiba Satellite 315CDT. Pentium-200 MMX, 64MB RAM. Working at a pretty good pace, considering.

**Comment by franck-888** [comment]
Skylake Intel(R) Core(TM) i5-6600 CPU @ 3.30GHz
Motherboard: Gigabyte Technology Co., Ltd. H170-HD3, BIOS F22e 03/09/2018
RAM: https://www.corsair.com/us/en/Categories/Products/Memory/VENGEANCE-LPX/p/CMK32GX4M2A2400C14#tab-tech-specs
RAM runs at DDR4-2133.
6.00b3 + git up to https://github.com/memtest86plus/memtest86plus/commit/8a3cac8133430b93129341c3c6105b7f1d3a5445
![skylake](https://user-images.githubusercontent.com/107723864/181591122-3d4e420b-7153-4945-846e-4c2a2e7707b9.jpg)

**Comment by dchard** [comment]
This is a Dell Inspiron 6420 with latest BIOS (Version 25) with the original Corei5 CPU upgraded to i7-2670QM.

![Memtest86_6beta3_dell_6420](https://user-images.githubusercontent.com/8152156/185704299-9f92b543-c27f-43e4-b8eb-eafd24561e2a.jpg)

It is especially pleasing to see that 4T testing works, as even the commercial version of Memtest86 latest version tells that _multi-core testing is not possible due to UEFI limitations_ on this laptop. 👍 

MOD: the reason for 4T is that I disabled HT previously. Now I retested it with HT enabled, and 8T tests also succeed.

**Comment by dchard** [comment]
And another successful test with a Ryzen 2600X in a Gigabyte Aorus M B450 (BIOS: F60):

![Memtest86_6beta3_gigabyte_B450](https://user-images.githubusercontent.com/8152156/185712179-46006776-16c7-4dee-a8c5-a96451ae0c3a.jpg)

**Comment by x86fr** [comment]
Tyan Tiger MP + Dual Athlon MP 1800+
![image](https://user-images.githubusercontent.com/38105886/218323435-39a8d5cd-066a-4b4f-a2ca-42ebf1aa64c7.jpeg)

Infamous PC Chips M507 with fake cache chips
![image](https://user-images.githubusercontent.com/38105886/218323453-d278010c-9d57-4cdd-a6d5-b7c81bc9073b.jpeg)

**Reply by debrouxl** [reply]
Interesting platforms. Looks like the former has an AMD766 SB, and as such, RAM SPD reporting would require new code paths in the SMBus controller driver.

**Reply by x86fr** [reply]
Unfortunately, many caps are bad and the platforms is very unstable. I'll try to recap the board soon.

**Comment by dchard** [comment]
HP MicroServer Gen8 (no EFI supported by BIOS):

![image](https://user-images.githubusercontent.com/8152156/219328897-009e662a-77ad-4343-b91f-066b1b3976c8.png)

Note: interestingly, grub can only be accessed with the virtual keyboard of the ILO HTML5 client (this is not a Memtest86+ issue, Ubuntu grub behaves the same). The remote (BMC) keyboard does not work. After passing grub, the remote keyboard starts to work.

**Comment by x86fr** [comment]
Asus P6NP5 😍
![image](https://user-images.githubusercontent.com/38105886/221423724-1281a9b4-ed09-4af7-b087-112c0951971f.jpeg)

**Comment by bjlockie** [comment]
![memtest-20230714_230107](https://github.com/memtest86plus/memtest86plus/assets/303873/3f90b37a-32c6-4e86-bac6-f41a23752779)

ASRock B450M Pro4

**Comment by birdie-github** [comment]
AMD Ryzen 7 5800X + DDR4 1800MHz dual channel, 4 modules

![memtest7](https://github.com/memtest86plus/memtest86plus/assets/4958603/bb4bea83-3103-4bc5-9051-2e9157ea33fe)

@x86fr 

Your 12600K results look abysmal in terms of memory bandwidth. Why is it so slooooooow? 14GBps? Really?

**Comment by birdie-github** [comment]
Raptor Lake 1360P

![rpl](https://github.com/memtest86plus/memtest86plus/assets/4958603/7e40a7e9-ae70-42d6-b041-a14536f93728)

**Comment by 01e3** [comment]
Dell Power Edge T110 II with E2-1230 V2, 4xSamsung M391B1G73QH0-CMA 8GB DDR3-1866 ECC, IMC supported.

![obraz](https://github.com/user-attachments/assets/2b301c67-0a24-49ae-8ac2-6819e213434f)

**Comment by dchard** [comment]
DL325 with single Epyc CPU:

![image](https://github.com/user-attachments/assets/a3126c38-9393-4905-b959-2a0c12e3433d)

SMP has to be disabled, otherwise the own address test takes forever

**Comment by jessicaPNW** [comment]
My results had a FAIL. Bought two 8 GB sticks of used RAM (Corsair Cmk16gx4m2b3000c15). Randomly remembered memory tests exist and decided to run memtest86+.

![20250429_091931~2](https://github.com/user-attachments/assets/78924a44-403c-44db-b3c9-c7f8048d8fff)
![20250429_091939~2](https://github.com/user-attachments/assets/f5f6d898-a480-497f-aca2-0270a370a9aa)
![20250429_092123~2](https://github.com/user-attachments/assets/9c6dcd81-28b1-416e-8174-3dd43488aa9d)

Edit: Added RAM make and model. Corsair CMK16GX4M2B3000C15.
